### PR TITLE
docker: Replay and cleanup scripts

### PR DESCRIPTION
**Pull request by yodapotatofly**
:warning: depends on #365

- create the cleanup script that wipes suricata logs and ES indices
- create the replay script to replay pcaps using  `suricata -r`
- move from ikev2 to ike in suricata (it was causing warnings)
- edit the customsuricata entrypoint in order to properly accept in-line arguments.

### CODE REVIEW

**Comment by pevma** [comment]
It might make more sense if we put all scripts in a scripts folder?

File: docker/replay.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    LOCAL_PATH="/replay/file.pcap"
+fi
+
+docker run --name suricata-replay --rm -it \
+--cap-add=net_admin --cap-add=sys_nice \
+-v ${BASEDIR}/containers-data/suricata/logs:/var/log/suricata \
+-v selks_suricata-rules:/etc/suricata/rules \
+-v ${BASEDIR}/containers-data/suricata/etc:/etc/suricata \
+-v ${HOST_PATH}:${LOCAL_PATH} \
+jasonish/suricata:master -r ${LOCAL_PATH} --runmode autofp -l /var/log/suricata --set sensor-name=suricata-replay
```
Comment (pevma) [inline docker/replay.sh:74]: add in please :
1- `-k none` 
2 - use `--runmode=single` by default
3 - open an issue to be able to supply if desired `--runmode=autofp`

Comment (pevma) [reply docker/replay.sh:74]: Lets try the following:
1- run in unix-socket mode   
2- pass the pcap via unix socket  (continuous mode will be added later as a possibility, lets use the default/easy mode first )

Separate issue:
if the cleanup script is used, make sure we send SIGHUP signal to the Suricata process

Comment (yodapotatofly) [reply docker/replay.sh]: > add in please : 1- `-k none` 2 - use `--runmode=single` by default 3 - open an issue to be able to supply if desired `--runmode=autofp`

all fixed

Comment (yodapotatofly) [reply docker/replay.sh]: > Separate issue: if the cleanup script is used, make sure we send SIGHUP signal to the Suricata process

done.

Comment (yodapotatofly) [reply docker/replay.sh]: > Lets try the following: 1- run in unix-socket mode 2- pass the pcap via unix socket (continuous mode will be added later as a possibility, lets use the default/easy mode first )

Let's make this a separate issue so that we can merge as it is for now ?

**Comment by yodapotatofly** [comment]
updated so that we can mix optional and positional arguments

File: docker/containers-data/suricata/etc/selks6-addin.yaml [hunk 2]
```diff
@@ -383,7 +383,7 @@ outputs:
         - nfs
         - smb
         - tftp
-        - ikev2
+        - ike
```
Comment (pevma) [inline docker/containers-data/suricata/etc/selks6-addin.yaml:386]: This should be in a separate commit as it has no relation to the reaply/clean up scripts.

Comment (yodapotatofly) [reply docker/containers-data/suricata/etc/selks6-addin.yaml]: you're right, done.
